scapy/contrib/isotp.py

```diff
@@ -1825,6 +1975,7 @@ def __init__(self,
             self.basecls = basecls
 
         def recv_raw(self, x=0xffff):
+            # type: (int) -> Tuple[Optional[Type[Packet]], Optional[bytes], Optional[float]]  # noqa: E501
```

Comment:
multiline

Reply:
here it's the return type, that fills the line.